pkg/internal/checkup/checkup.go

```diff
@@ -69,8 +70,11 @@ type Checkup struct {
 }
 
 const (
-	VMINamePrefix                 = "dpdk-vmi"
-	TrafficGeneratorPodNamePrefix = "kubevirt-dpdk-checkup-traffic-gen"
+	VMINamePrefix                       = "dpdk-vmi"
+	TrafficGeneratorPodNamePrefix       = "kubevirt-dpdk-checkup-traffic-gen"
+	appLabelKey                         = "app"
+	VMIAppLabelValuePrefix              = "testpmd-"
+	TrafficGeneratorAppLabelValuePrefix = "trex-"
```

Comment:
Could you please elaborate why are these necessary?
AFAIU, we only need a single label to create pod anti-affinity.

Reply:
yep removed.